localization/ndt_scan_matcher/include/ndt_scan_matcher/ndt_scan_matcher_core.hpp

```diff
@@ -134,6 +137,7 @@ class NDTScanMatcher : public rclcpp::Node
     regularization_pose_sub_;
 
   rclcpp::Publisher<sensor_msgs::msg::PointCloud2>::SharedPtr sensor_aligned_pose_pub_;
+  rclcpp::Publisher<sensor_msgs::msg::PointCloud2>::SharedPtr no_ground_aligned_pose_pub_;
```

Comment:
Sorry. I misinterpreted the variable name.  Perhaps you named it based on `sensor_aligned_pose_pub_`, right? 
It is hard to know what it publishes, so it would be better to include "points" in the publisher name.

Reply:
OK! thanks.